jupyterhub_config.py

```diff
@@ -36,7 +36,8 @@
 # Mount the real user's Docker volume on the host to the notebook user's
 # notebook directory in the container
 c.DockerSpawner.volumes = { 'jupyterhub-user-{username}': notebook_dir }
-c.DockerSpawner.extra_create_kwargs.update({ 'volume_driver': 'local' })
+# volume_driver is no longer a keyword argument to create_container()
+# c.DockerSpawner.extra_create_kwargs.update({ 'volume_driver': 'local' })
```

Comment:
Why not just deleting the line?

Reply:
@dalg24 fair point, it did cross my mind, but (a) this might still be necessary for people to keep if they are running an older version of docker-py (b) it documents the usage of `extra_create_kwargs`. Your call, feel free to send another PR.